spec/models/user_spec.rb

```diff
@@ -23,5 +24,17 @@
       review_2 = @book_2.reviews.create!(title: "This book sucks!", rating: 1, text: "Don't read it!", user: @user_1)
       expect(@user_1.reviews.count).to eq(2)
     end
+
+    it "sorts reviews by date" do
+      travel 1.day
+      review_1 = @book_1.reviews.create!(title: "This book rocks!", rating: 5, text: "Read it!", user: @user_1)
+      travel_back
+      review_2 = @book_2.reviews.create!(title: "This book sucks!", rating: 1, text: "Don't read it!", user: @user_1)
+      travel 2.day
+      review_3 = @book_3.reviews.create!(title: "This book is OK.", rating: 3, text: "Read it if you want.", user: @user_1)
+
+      expect(@user_1.sorted_reviews("date-desc").to_a).to eq([review_3, review_1, review_2])
+      expect(@user_1.sorted_reviews("date-asc").to_a).to eq([review_2, review_1, review_3])
```

Comment:
We should add a third test here for a non-supported sort_order argument.

Reply:
Agreed, adding test on branch.